src/fetch_agent/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>fetch_agent</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="Tobias.frichmann1@gmail.com">tobinio</maintainer>
```

Comment:
Possibility for using system variables like $user from windows?

Reply:
for the maintainer?

i think we should just look into adding multiple